Common/EndToEnd.Tests/Tests/CheckRandomException/RandomExceptionSym.cs

```diff
@@ -101,17 +101,23 @@ private double ExpM1(double x)
     public override byte[] OnInvoke(SessionContext sessionContext, TaskContext taskContext)
     {
       var clientPayload = ClientPayload.Deserialize(taskContext.TaskInput);
+      using var _ = Logger.BeginScope("OnInvoke Random",
+                                      ("SessionId", sessionContext.SessionId),
+                                      ("TaskId", taskContext.TaskId),
+                                      ("ClientPayload", clientPayload.Type));
 
+      Logger.LogInformation("{ClientPayload} task, sessionId : {SessionId}, taskId : {TaskId}",
```

Comment:
As far as I'm aware, `SessionId`, `TaskId` and `ClientPayload` are automatically added to any logs because of `Logger.BeginScope`. I don't think it is necessary to add them again here.
But you most likely still need the call to `Logger.LogInformation`.

Reply:
Client uses it to find from message in seq but also in kibana

Reply:
My point is: I think the information is duplicated because it is already included thanks to `Logger.BeginScope`. I'm fine with keeping it this way.